gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/KeelService.java

```diff
@@ -68,4 +68,12 @@ Map getApplicationDetails(
 
   @GET("/vetos/{name}/rejections")
   List<String> getVetoRejections(@Path("name") String name);
+
+  @GET("/export/{cloudProvider}/{account}/{type}/{name}")
+  Resource exportResource(
```

Comment:
Is this the right return type?

Reply:
Yeah, the export controller in keel specifically returns a `SubmittedResource` which only differs from a `Resource` in that the latter has additional metadata validation. No distinction from gate's perspective though.

Reply:
great!